doc/cluster-management/olvm.md

```diff
@@ -0,0 +1,546 @@
+**NOTE: This is a developer release**
+
+# OLVM Cluster API Provider
+The Oracle Linux Virtualization Manager (OLVM) Cluster API Provider allows you to create Kubernetes clusters on an
+existing OLVM instance. The cluster nodes can be spread
+across multiple OLVM hosts, where both the control plane and worker nodes can
+be scaled in and out as desired. Using the Oracle Cloud Native Environment CLI (`ocne`), 
```

Comment:
Is it ok to use `ocne`?  I thought we were not supposed to.

Reply:
that is the CLI, i define it up above.